src/commands/core/collection/create.ts

```
@@ -3,6 +3,8 @@ import { generateSigner, PublicKey, Umi } from '@metaplex-foundation/umi'
 import { Flags } from '@oclif/core'
 import fs from 'node:fs'
 import ora from 'ora'
+import { mintKeypairFlag } from '../../../lib/mintKeypair.js'
```

ESLint

[error] Missed spacing between "ora" and "../../../lib/mintKeypair.js" imports

(perfectionist/sort-imports)